components/media-player/MediaPlayer.vue

```diff
@@ -25,7 +25,13 @@ defineShortcuts({
     :class="{ 'w-[400px]': open, 'w-0': !open }"
   >
     <div
-      class="large-shadow group fixed bottom-1 right-0 z-30 flex w-[400px] max-w-full animate-[player-enter_200ms] flex-col overflow-hidden rounded-2xl bg-background-1 transition-all duration-700 ease-out-expo sm:bottom-5 sm:right-5"
+      :class="[
+        'large-shadow group fixed bottom-1 right-0 z-30 flex w-[400px] max-w-full animate-[player-enter_200ms] flex-col overflow-hidden bg-background-1 transition-all duration-700 ease-out-expo sm:bottom-5 sm:right-5',
+        {
+          'rounded-[32px]': open,
+          'rounded-2xl': !open,
```

Comment:
We now have two different border radii on closed and open states, as in Figma.
Should probably not use `[32px]` like this, though.